charts/model-engine/templates/inference_framework_config.yaml

```diff
@@ -2,17 +2,18 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ include "modelEngine.fullname" . }}-inference-framework-latest-config
+  namespace: {{ .Release.Namespace }}
   labels:
     product: common
     team: infra
   annotations:
-    "helm.sh/hook": pre-install
+    "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-weight": "-2"
 data:
   deepspeed: "latest"
   text_generation_inference: "latest"
-  vllm: "latest"
-  vllm_batch: "latest"
-  vllm_batch_v2: "latest"
+  vllm: "{{ .Values.vllmTag }}"
+  vllm_batch:  "{{ .Values.vllmTag }}"
+  vllm_batch_v2:  "{{ .Values.vllmTag }}"
```

Comment:
nice to specify

Reply:
these are different images so we would want vllmTag, vllmBatchTag, vllmBatchV2Tag; and yes let's default to latest

Reply:
done